airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/check/commands.py

```diff
@@ -30,21 +43,43 @@ async def all_languages(ctx: click.Context, list_errors: bool):
     """
     Run all format checks and fail if any checks fail.
     """
+    commands_to_invoke = [command for command_name, command in check.commands.items() if command_name != ctx.command.name]
```

Comment:
Did we forget to do this same thing for fix?

Reply:
On the note of fix lets make sure to reuse the logic

Reply:
Yes I just changed `check` yet...